app/subscriber/src/components/tool-bar/AddToReportMenu.tsx

```diff
@@ -20,6 +20,7 @@ export const AddToReportMenu: React.FC<IAddToReportMenuProps> = ({ content, onCl
   const [{ requests }] = useApp();
   const [activeReport, setActiveReport] = React.useState<IReportModel>();
   const [reportId, setReportId] = React.useState<number | null>(null);
+  const [inProgress, setInProgress] = React.useState({ sectionName: '', value: false });
```

Comment:
So we can display a loading indicator next to the section that the user is adding content to